dali/operators/random/rng_base_gpu.h

```diff
@@ -27,37 +27,53 @@
 
 namespace dali {
 
-struct BlockDesc {
+
+template <bool NeedsInput>
+struct BlockDesc;
+
+template <>
+struct BlockDesc<false> {
   int sample_idx;
   void* start;
```

Comment:
Why not `output` to be consistent with `BlockDesc<true>`.

Reply:
I can do that.